src/components/SliderWithInput.jsx

```diff
@@ -14,11 +14,21 @@ const SliderWithInput = (props) => {
   const [, handleChange] = useUpdateThrottled(onUpdate, value);
 
   const [localValue, setLocalValue] = useState(value);
+  const [writtenLocalValue, setWrittenLocalValue] = useState(value);
 
   useEffect(() => {
-    setLocalValue(value);
+    setLocalValue(parseFloat(value));
   }, [value]);
 
+  useEffect(() => {
```

Comment:
These changes were a minor change of behavior to have a delayed run of `handleChange` after writing numbers in the input box, unrelated to the main issue addresed in the ticket.